robottelo/ui/navigator.py

```diff
@@ -12,6 +12,7 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                    tertiary_menu_locator=None):
         self.perform_action_chain_move(top_menu_locator)
         if not tertiary_menu_locator:
+            self.wait_until_element(sub_menu_locator)
```

Comment:
do not see why we have to call explicitly  wait_until_element here as already called in click function

Reply:
Might have been a problem with my test-setup. I tested with firefox, but as I learned recently there seem to be problems with firefox not waiting for elements.
I can remove this line, if you like.